### PR TITLE
Designing Index and show pages for Teams, Users, & Tags.

### PR DESCRIPTION
**Pull request by Langster2323**

### CODE REVIEW

File: app/views/tags/show.html.erb [hunk 1]
```diff
@@ -1,2 +1,6 @@
-<h3>Tag</h3>
-<div>Name: <%= tag.name %></div>
+<div class="card file-card">
+  <div class="card-content">
+    <h3 class="card-title">Tag</h3>
+    <div>Name: <%= tag.name %></div>
```
Comment (MattRice12) [inline app/views/tags/show.html.erb]: take out 'Name: '

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -6,21 +6,29 @@
 
   <% user.photos.each do |photo| %>
     <% if has_permission?(photo) %>
-      <h5><%= link_to photo.name, photo %></h5>
-      <p>Category: <%= link_to photo.category.name, photo.category %></p>
+  <div class="card file-card">
+    <div class="card-content">
+      <h5 class="card-title"><%= link_to photo.name, photo %></h5>
```
Comment (MattRice12) [inline app/views/users/show.html.erb]: you need an image icon here, too

File: app/controllers/teams_controller.rb [hunk 3]
```diff
@@ -26,11 +26,11 @@ def create
...
-      if team.save
-        redirect_to team
-      else
-        render :new, locals: { team: team }
-      end
+    end
+    if team.save
+      redirect_to team
+    else
+      render :new, locals: { team: team }
```
Comment (rluck0419) [inline app/controllers/teams_controller.rb]: I'm not sure I understand what's going on here. I don't think it was working before either though - I think lines 30-33 should be between line 26 & 27.

**Comment by rluck0419** [comment]
after the teams_controller fix, looks good to merge 👍

**Pull request merged**